File: Basic-Video-Chat-Java/app/src/main/java/com/tokbox/sample/basicvideochat/network/GetSessionResponse.java
package com.tokbox.sample.basicvideochat.network;

import com.squareup.moshi.Json;

public class GetSessionResponse {
    @Json(name = "apiKey") public String apiKey;
    @Json(name = "sessionId") public String sessionId;
    @Json(name = "token") public String token;
}

File: Basic-Video-Chat-Kotlin/app/src/main/java/com/tokbox/sample/basicvideochat/network/GetSessionResponse.kt
package com.tokbox.sample.basicvideochat.network

import com.squareup.moshi.Json

class GetSessionResponse {

    @Json(name = "apiKey")
    var apiKey: String = ""

    @Json(name = "sessionId")
    var sessionId: String = ""

    @Json(name = "token")
    var token: String = ""
}